modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -466,6 +469,15 @@ public void setFileIOFactory(FileIOFactory ioFactory) {
                 "write ahead log archive directory"
             );
 
+            if (dsCfg.isCdcEnabled()) {
+                cdcDir = initDirectory(
+                    dsCfg.getCdcWalPath(),
+                    DataStorageConfiguration.DFLT_WAL_CDC_PATH,
+                    resolveFolders.folderName(),
+                    "cdc directory"
```

Comment:
as you may see, some lines before, even WAL presented without using the abbreviation %)

Reply:
> even WAL presented without using the abbreviation

I think we should fix this terrible error!!!